src/features/cartographie/components/lieux-mediation-numerique-details/accessibilite/accessibilite.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="bg-light p-2 d-print-none d-flex">
+  <div class="d-flex flex-grow-1 mt-3">
+    <div class="d-flex" *ngIf="accessibilite; else noAccessibilite">
+      <a href="{{ accessibilite }}" class="btn btn-sm btn-secondary d-block mb-2" target="_blank" rel="noopener noreferrer">
+        <i class="ri-wheelchair-line me-2 text-right"></i>
+        <span>Consulter l'accessibilité</span>
+      </a>
+      <div class="ps-3 d-flex mt-1 text-primary">
+        <i class="ri-check-line me-1"></i>
+        <p>Informations renseignées</p>
+      </div>
+    </div>
+    <ng-template #noAccessibilite>
+      <a
+        href="https://acceslibre.beta.gouv.fr/contrib/start/"
+        class="btn btn-sm btn-secondary d-block mb-2"
+        target="_blank"
+        rel="noopener noreferrer">
+        <i class="ri-edit-line me-2 text-right"></i>
+        <span>Renseigner l'accessibilité</span>
+      </a>
+      <div class="ps-3 d-flex mt-1 text-info">
```

Comment:
```suggestion
      <div class="ps-3 d-flex mt-1 text-muted-dark">
```
Le texte est trop clair par rapport au proto